### ISSUE TITLE
styling the input text field and button 

### ISSUE THREAD

**Issue by tundeaji1232** [fac-12/FlagFinder#27]
really cool website....though the nigerian flag didn't come out as sexy as i thought.... :smile: 

Abit more styling of the submit button and maybe sizing the input field better would make it look super cool..

**Comment by Johanna-hub** [comment]
I agree that the input field needs to be bigger and a bigger font to be easier to read
